src/test/data/JsonSerializableAddressBookTest/typicalClientsAddressBook.json

```diff
@@ -8,7 +8,16 @@
     "country" : "SG",
     "timezone": "GMT+8",
     "contractExpiryDate": "1-4-2021",
-    "tagged" : [ "friends" ]
+    "tagged" : [ "friends" ],
+    "clientNotes" : [ {
+      "contents" : "lives in a pineapple",
+      "countryCode" : "NULL_CC",
```

Comment:
actually for client notes, is it possible to not store the countryCode in the first place? Seems a bit wasteful and unneeded. JsonCreator and toModelType() can probably handle the lack of the countryCode programatically. @raysonkoh any comments about this?

Reply:
Will create an issue for this. Currently it is working and it is quite close to end of v1.3, changing this might introduce unintentional bugs.

Reply:
@raysonkoh I think if we change it, we should change before the end of v1.3, since we have our bug-finding practice coming up and people can help catch the bugs if any. If we change after v1.3, we might introduce some bugs that no one finds until the final bug-finding test.